releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -4,6 +4,11 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v3.5.16 - July, 22 2021
+* Added the capability to use certificate based authentication.
+* Added logs that show more information about the task.
```

Comment:
The logs we added are just for the parameters being used to run the task right? If so, can we call that out specifically?

Reply:
Yes